lostpixel.config.ts

```diff
@@ -0,0 +1,30 @@
+import { CustomProjectConfig, PageScreenshotParameter } from "lost-pixel";
+
+const defaultPageOptions: Partial<PageScreenshotParameter> = {
+    waitBeforeScreenshot: 2500
+}
+
+export const config: CustomProjectConfig = {
+    pageShots: {
+        pages: [
```

Comment:
Would be really cool if we found a way to generate the page list for your DS, recently we added support for programmatic page runs meaning you could generate the array, put it into json and host it statically on your ran website e.g. localhost:3000/pagesToTestWithLP.json and we just pick the info from there ✌🏼